Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM alpine:3.13
+ARG ARCH
+FROM ${ARCH}alpine:3.13
+
+ARG K0S_VERSION
 
 RUN apk add --no-cache bash coreutils findutils iptables curl tini
```

Comment:
I don't think we need iptables any more.

Reply:
Removed iptables